cnf-tests/pod-utils/cyclictest-runner/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"k8s.io/klog"
+	"os/exec"
+	"strconv"
+	"time"
+
+	podutils "github.com/openshift-kni/cnf-features-deploy/cnf-tests/pod-utils"
+)
+
+const cyclictestBinary = "/usr/bin/cyclictest"
+
+func main() {
+	klog.InitFlags(nil)
+
+	var rtPriority = flag.String("rt-priority", "1", "specify the SCHED_FIFO priority (1-99)")
```

Comment:
No need to use var, you can use the short declaration form here rtPriority := ..